service/ui.go

```diff
@@ -275,6 +278,10 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 		))
 	}
 
+	// GraphQL
+	app.AddRoute("/graphql").Handler(playground.Handler("GraphQL playground", "/graphql/query")).Options().Post().Get()
+	app.AddRoute("/graphql/query").Handler(handler.NewDefaultServer(graphql.NewExecutableSchema(graphql.New())).ServeHTTP).Options().Post().Get()
```

Comment:
we should probably restrict access to logged in users here.

Reply:
adding authentication is another ticket

Reply:
we can't deploy this until you add authentication to this.

add `.Wrap(needsLogin)` to the chain for these results.

Reply:
added